debugger/static/cpu.js

```diff
@@ -0,0 +1,18 @@
+class CPU extends HTMLUListElement {
+    constructor() {
+        super();
+        this.id = "cpu";
+    }
+
+    addState(cpu) {
+        const li = document.createElement("li");
+        li.appendChild(document.createTextNode(Object.keys(cpu).map(key => `${key}: ${cpu[key]}`).join(',')));
+        this.appendChild(li);
+    }
+}
+
+customElements.define('cpu-div', CPU, {extends: 'ul'});
```

Comment:
nit: can we use a constant for the name of the component ?

Reply:
good idea